### PR TITLE
Removed stdiotell.StdIOWrapper

### PR DESCRIPTION
**Pull request by claudep**
I may have missed something, but I didn't find any useful usage for the stdiotell.StdIOWrapper wrapper class in the current version of the ttk.

### CODE REVIEW

File: translate/convert/tiki2po.py [hunk 1]
```diff
@@ -80,8 +80,6 @@ def converttiki(inputfile, outputfile, template=None, includeunused=False):
 def main(argv=None):
     """Converts tiki .php files to .po."""
     from translate.convert import convert
-    from translate.misc import stdiotell
-    sys.stdout = stdiotell.StdIOWrapper(sys.stdout)
```
Comment (unho) [inline translate/convert/tiki2po.py]: You forgot to remove `sys` import.

**Comment by unho** [comment]
I am a bit concerned that we might be breaking something by removing `stdiotell.StdIOWrapper`, but it seems harmless and tests seem to pass. So will merge after the nit is addressed and the PR is rebased.

**Comment by claudep** [comment]
I'm almost certain we don't break anything in TTK itself, as I've audited all `tell()` calls. Of course, we never know about third-party code, but I don't see the real added value in that utility. It was certainly useful in the past for some usage.
Removed the leftover import and rebased.

**Comment by unho** [comment]
Merged. Thanks!

**Pull request merged**